python/activator/middleware_interface.py

```diff
@@ -905,12 +905,36 @@ def _prepend_collection(butler: Butler, chain: str, new_collections: collections
         The butler in which the collections exist.
     chain : `str`
         The chained collection to prepend to.
-    new_collections : iterable [`str`]
-        The collections to prepend to ``chain``.
+    new_collections : sequence [`str`]
+        The collections to prepend to ``chain``, in order.
 
     Notes
     -----
     This function is not safe against concurrent modifications to ``chain``.
     """
     old_chain = butler.registry.getCollectionChain(chain)  # May be empty
     butler.registry.setCollectionChain(chain, list(new_collections) + list(old_chain), flatten=False)
+
+
+def _remove_from_chain(butler: Butler, chain: str, old_collections: collections.abc.Iterable[str]) -> None:
+    """Remove a specific collection from a chain.
+
+    This function has no effect if the collection is not in the chain.
+
+    Parameters
+    ----------
+    butler : `lsst.daf.butler.Butler`
+        The butler in which the collections exist.
+    chain : `str`
+        The chained collection to prepend to.
```

Comment:
The chained collection to _remove from_.

Reply:
I could swear I fixed that... 🤦